### ISSUE TITLE
Reproducibility Issue: Training produces near-zero validation Dice, despite decreasing training loss (compared to working pre-trained model)

### ISSUE THREAD

**Issue by afroozsheikh** [Jupitern52/TextBraTS#2]
I am encountering a critical issue when attempting to reproduce the training results of the TextBraTS repository. I have meticulously followed all instructions, including using the provided dataset and the --use_ssl_pretrained flag for training.

The core problem is a complete disconnect between training loss and validation performance:
During training, the training loss consistently decreases from approximately 0.98 to 0.1 over 150 epochs, indicating that the model is learning something. However, the mean validation Dice score remains extremely low, hovering around 0.011 to 0.012 throughout the entire training process.

Crucially, when I evaluate the model I trained on the 93 test samples, I obtain all zero metrics:

Final Validation Results - Dice_TC: 0.0, Dice_WT: 0.0, Dice_ET: 0.0, Avg Dice: 0.0, HD95_TC: 0.0, HD95_WT: 0.0, HD95_ET: 0.0, Avg HD95: 0.0

I have meticulously verified the following points:

Data and Preprocessing: I am using the exact dataset provided by the authors. I have carefully reviewed and confirmed that the preprocessing steps (normalization, resizing, channel order, etc.) implemented in my setup are identical to those specified in the repository's code, for both training and validation data.

SSL Pre-training: The --use_ssl_pretrained flag is correctly utilized, and the pre-trained weights are loaded without error messages.

Code Integrity: I am running the main.py script directly from the cloned repository without any modifications to the core training, validation, loss, or metric calculation logic.

My Environment:

Operating System: Ubuntu 22.04.5 
Python Version: Python 3.10.12
PyTorch Version: 2.7.1+cu128
CUDA Version: 12.8
GPU: NVIDIA RTX 5090

pip freeze output:

absl-py==2.3.1
einops==0.8.1
filelock==3.18.0
fsspec==2025.5.1
grpcio==1.73.1
importlib_resources==6.5.2
Jinja2==3.1.6
Markdown==3.8.2
MarkupSafe==3.0.2
monai==1.5.0
mpmath==1.3.0
networkx==3.4.2
nibabel==5.3.2
numpy==2.2.6
nvidia-cublas-cu12==12.8.3.14
nvidia-cuda-cupti-cu12==12.8.57
nvidia-cuda-nvrtc-cu12==12.8.61
nvidia-cuda-runtime-cu12==12.8.57
nvidia-cudnn-cu12==9.7.1.26
nvidia-cufft-cu12==11.3.3.41
nvidia-cufile-cu12==1.13.0.11
nvidia-curand-cu12==10.3.9.55
nvidia-cusolver-cu12==11.7.2.55
nvidia-cusparse-cu12==12.5.7.53
nvidia-cusparselt-cu12==0.6.3
nvidia-nccl-cu12==2.26.2
nvidia-nvjitlink-cu12==12.8.61
nvidia-nvtx-cu12==12.8.55
packaging==25.0
pillow==11.0.0
protobuf==6.31.1
scipy==1.15.3
six==1.17.0
sympy==1.13.3
tensorboard==2.19.0
tensorboard-data-server==0.7.2
torch==2.7.1+cu128
torchaudio==2.7.1+cu128
torchvision==0.22.1+cu128
tqdm==4.67.1
triton==3.3.1
typing_extensions==4.14.1
Werkzeug==3.1.3

**Comment by Jupitern52** [comment]
Thank you for reporting this. I believe your issue is related to the same MONAI compatibility problem I described in this issue. To keep things organized, let’s continue the discussion there. Thanks for your understanding!
